content/reference/plugins/gitlab_ci/index.md

```diff
@@ -22,55 +32,193 @@ In the following examples, we assume the API Token is available in an environmen
+
+```yaml
+stages:
+  - load-test
+  - post-load-test
+
+run-gatling-enterprise:
+  stage: load-test
+  image:
+    name: guilgaly/enterprise-runner:latest
+    entrypoint: ['']
+  script:
+    - gatlingEnterpriseStart
+  variables:
+    SIMULATION_ID: '00000000-0000-0000-0000-000000000000'
+  artifacts:
+    reports:
+      dotenv: 'gatlingEnterprise.env' # Using the default value
+
+print-output:
+  stage: post-load-test
+  image: alpine:latest
+  script: |
+    # Show that we can access the outputs exported by the previous stage
+    echo "run_id=$run_id"
+    echo "reports_url=$reports_url"
+    echo "runs_url=$runs_url"
+    echo "runs_status_code=$runs_status_code"
+    echo "runs_status_name=$runs_status_name"
+    echo "run_assertions=$run_assertions"
+```
+
+- `run_id`: The ID of the run started by this action.
+
+- `reports_url`: The URL of the reports page for this run.
+
+- `runs_url`: The URL of the runs history page for this simulation.
+
+- `runs_status_name`: The name of the run's final status (e.g. `Successful`, `AssertionsSuccessful`, `AssertionsFailed`, etc.).
+
+- `runs_status_code`: The code of the run's final status.
+
+- `run_assertions`: The results of the run's assertions, as a JSON array.
+
+### Logs
+
+Every few seconds, the Docker runner logs to the console output a summary of the run's current status. When the run ends, it logs the status of the run and the results of any assertions. Here's the beginning and end of a very short duration example:
+
+{{< img src="reference_logs_start.png" alt="A run's logs in the GitLab CI/CD console (beginning)" >}}
+
+{{< img src="reference_logs_end.png" alt="A run's logs in the GitLab CI/CD console (end)" >}}
+
+## Sample use cases
+
+### Build and run simulation
+
+This pipeline is defined in the GitLab repository which contains your Gatling simulation script built with one of our build tools plugins. In this example, every time the code on the `main` branch gets updated, we:
+
+- build, package, and upload to Gatling Enterprise the current version of the simulation script
+- run the updated simulation on Gatling Enterprise
+
+Feel free to use different trigger events or to configure the other inputs and outputs for the Action as documented above, according to your own use case.
+
+{{< include-file >}}
+Maven: includes/use-case-build-and-run.maven.md
+Gradle: includes/use-case-build-and-run.gradle.md
+Gradle Wrapper: includes/use-case-build-and-run.gradlew.md
+Sbt: includes/use-case-build-and-run.sbt.md
+{{< /include-file  >}}
+
+### Build and update on every push, run on a schedule
+
+This first pipeline is defined in the GitLab repository which contains your Gatling simulation script built with one of our build tools plugins. In this example, every time the code on the `main` branch gets updated, we build, package, and upload to Gatling Enterprise the current version of the simulation script.
+
+{{< include-file >}}
+Maven: includes/use-case-separate-build-run-1.maven.md
```

Comment:
If you add gradle wrapper, you should add maven wrapper as well.

Reply:
I had mostly included the Gradle wrapper examples because it seems to be very widely used in the Gradle ecosystem (and it should be, considering Gradle's versions incompatibilities...), I'm not sure Maven Wrapper is that commonly used? But I can always add it.

Reply:
Please do.

Reply:
Done.